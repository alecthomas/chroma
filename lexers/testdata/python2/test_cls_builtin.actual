class TestClass():
    @classmethod
    def hello(cls):
        pass
