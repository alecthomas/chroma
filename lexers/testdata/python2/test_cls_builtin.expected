[
  {"type":"Keyword","value":"class"},
  {"type":"Text","value":" "},
  {"type":"NameClass","value":"TestClass"},
  {"type":"Punctuation","value":"():"},
  {"type":"Text","value":"\n    "},
  {"type":"NameDecorator","value":"@classmethod"},
  {"type":"Text","value":"\n    "},
  {"type":"Keyword","value":"def"},
  {"type":"Text","value":" "},
  {"type":"NameFunction","value":"hello"},
  {"type":"Punctuation","value":"("},
  {"type":"NameBuiltinPseudo","value":"cls"},
  {"type":"Punctuation","value":"):"},
  {"type":"Text","value":"\n        "},
  {"type":"Keyword","value":"pass"},
  {"type":"Text","value":"\n"}
]
